lib/active_merchant/billing/gateways/mercado_pago.rb

```diff
@@ -196,6 +198,46 @@ def add_payment(post, options)
         post[:payment_method_id] = options[:payment_method_id] if options[:payment_method_id]
       end
 
+      def add_net_amount(post, options)
+        post[:net_amount] = options[:net_amount].to_f if options[:net_amount]
+      end
+
+      def add_taxes(post, options)
+        return unless (tax_object = options[:taxes])
+
+        if tax_object.is_a?(Array)
+          post[:taxes] = process_taxes_array(tax_object)
+        elsif tax_object.is_a?(Hash)
+          post[:taxes] = process_taxes_hash(tax_object)
+        else
+          raise taxes_error
+        end
+      end
+
+      def process_taxes_hash(tax_object)
+        [sanitize_taxes_hash(tax_object)]
+      end
+
+      def process_taxes_array(taxes_array)
+        taxes_array.map do |tax_object|
+          raise taxes_error unless tax_object.is_a?(Hash)
+          sanitize_taxes_hash(tax_object)
+        end
+      end
+
+      def sanitize_taxes_hash(tax_object)
+        tax_value = tax_object['value'] || tax_object[:value]
+        tax_type = tax_object['type'] || tax_object[:type]
+
+        raise taxes_error if tax_value.nil? || tax_type.nil?
+
+        { value: tax_value.to_f, type: tax_type }
+      end
+
+      def taxes_error
+        ArgumentError.new("Taxes should be a single object or array of objects with the shape: { value: 500, type: 'IVA' }")
```

Comment:
Minor nitpick to add "following" before the colon here:

```suggestion
        ArgumentError.new("Taxes should be a single object or array of objects with the following shape: { value: 500, type: 'IVA' }")
```